x-pack/plugins/observability_onboarding/public/components/app/system_logs/install_elastic_agent.tsx

```diff
@@ -60,11 +58,10 @@ export function InstallElasticAgent() {
     navigateToKibanaUrl('/app/observabilityOnboarding');
   }
   async function onContinue() {
-    const dataStreams = getSystemLogsDataStreams();
-    const dataSets = dataStreams.map(
-      (dataSream) => dataSream.data_stream.dataset
-    );
-    await locator?.navigate(getDiscoverNavigationParams(dataSets));
+    await locators.singleDatasetLocator.navigate({
```

Comment:
don't we need here the multiDatasetLocator? In the [configuration](https://github.com/elastic/kibana/blob/df08786c98cd12e020934126e8d6af3f460d0a34/x-pack/plugins/observability_onboarding/common/elastic_agent_logs/system_logs/generate_system_logs_yml.ts#L69) we are creating two datasets `syslog` and `auth`